dev-support/docker/Dockerfile

```diff
@@ -97,9 +94,7 @@ RUN apt-get -q install --no-install-recommends -y zulu-8 zulu-7
 RUN update-alternatives --config java
 RUN update-alternatives --config javac
 RUN mv /usr/lib/jvm/java-7-openjdk-amd64 /usr/lib/jvm/moved.java-7-openjdk-amd64
-RUN mv /usr/lib/jvm/zulu-7-amd64 /usr/lib/jvm/java-7-openjdk-amd64 
-RUN mv /usr/lib/jvm/zulu-8-amd64 /usr/lib/jvm/java-8-openjdk-amd64 
-
+ENV JAVA_HOME /usr/lib/jvm/zulu-7-amd64
```

Comment:
Rather than manage the variable in Docker, why not pass it in as `--java-home` ?

Reply:
Can't do that because in the case of the general precommit job (the one that looks at patches on Jira) we don't know what branch we're targeting until Yetus is running.

Without knowing what branch we're on we can't know which jdk to point to.

Yetus will then take care of getting the Dockerfile specific to that branch.

Reply:
Perhaps a side-thought, but it seems like a concept of a `yetusrc` file would be useful -- a set of constants/declarations that can be committed to each branch. Or maybe that's not helpful because these values vary according to conditional logic as expressed in the `Jenkinsfile`? My somewhat selfish objective would be to have the `Jenkinsfile` definitions as small as possible, such that one could run as much of the pre-commit pipeline as possible from their local workstation, without magical Jenkins context.